### DISCUSSION TITLE
vant-taro 组件库

### DISCUSSION THREAD

**Discussion by ickg5** [youzan/vant#9314] [Show and tell]
项目地址：https://gitee.com/mallfoundry/taroify 
文档地址：https://taroify.gitee.io/taroify.com/introduce/

一个采用与 vant 相同视觉的 taro 组件库。

**Comment by chenjiahan** [comment]
👏 近期会添加到文档中，期待继续维护和完善喔

**Comment by ghost** [comment]
@tgioer  为啥不发布到github 里 哈哈 我在github 找了半天 没有找到地址

**Comment by ickg5** [comment]
https://github.com/mallfoundry/taroify
我有的时候会同步

**Comment by chenjiahan** [comment]
文档已添加✌️
